README.md

```diff
@@ -37,3 +37,16 @@ $ npm run dev
 - If any API changes have been made:
     - Write JS Doc on top of your routes using YAML based annotations in OPEN API 3.0 format.
     - Run `npm run generate-api-schema` to generate the API schema. A file `public/apiSchema.json` will be created/updated.
+
+## Getting the cookie locally
+- Run the project locally, make sure the server is listening to requests
+- Open a new tab in the browser
+- Open the `Network` tab in Browsers Developer Tools
+- Make sure the `Preserve log` checkbox is selected  
+  ![Preserve logs checkbox](https://user-images.githubusercontent.com/26569942/102855521-cea32880-444a-11eb-84e3-02ee6fa5b830.png)
+- Put the following URL in browsers URL bar with the clientId you got from GitHub
+    <br />
+    `https://github.com/login/oauth/authorize?client_id=<clientId>`
+- The link will take you to the GitHub login flow and it will redirect to registered callback url after authentication
+- In the browsers Dev tools Network tab, look for `http://localhost:3000/auth/github/callback?code=<SOME_AUTOGENERATED_CODE>` request. This is the response that actually sets the cookie on the website, but because we made the call from localhost, it will not be honoured by the browser, but we can manually copy the cookie from **Response Headers** and use it for our Postman and Swagger calls.  
```

Comment:
Can we simply add the URL `<HOST>/auth/github/callback?code=<SOME_AUTOGENERATED_CODE>` in the third point here https://github.com/Real-Dev-Squad/website-backend/blob/develop/CONTRIBUTING.md#generating-authentication-token to make it more explicit?

Reply:
Sure, I think it'll be better if we move all this to the `CONTRIBUTION.md`.
I'll make the changes